docs/content/modules/products/admin/manage-categories.mdx

```diff
@@ -157,7 +157,7 @@ const CreateCategory = () => {
 
   const handleCreate = () => {
     createCategory.mutate({
-      name: "Skinny Jeans",
+      name: "Skinny Jeans"
```

Comment:
The fix should be made in the curl example instead (line 198)

Reply:
Ups, fixed.